gallery/visualtests/darkModeSettings.js

```diff
@@ -0,0 +1,59 @@
+/*
+ * Copyright (c) 2019-present Sonatype, Inc.
+ * This program and the accompanying materials are made available under
+ * the terms of the Eclipse Public License 2.0 which accompanies this
+ * distribution and is available at https://www.eclipse.org/legal/epl-2.0/.
+ */
+
+const { setupBrowser } = require('./testUtils');
+
+describe('Dark Mode Settings', function() {
+  const { getPage,
+    checkFullPageScreenshot,
+    setThemeOverride,
+    setThemingEnabled
+  } = setupBrowser('#/');
+
+  async function themeOverrideTest(theme) {
```

Comment:
To be more succinct and more consistent with our utilities like `simpleTest`, `themeOverrideTest` should be a non-async function which returns an anonymous async function which runs the actual test logic.  Then the `it` statements would just be `it('...', themeOverrideTest(...));`

Reply:
> To be more succinct and more consistent with our utilities like `simpleTest`, `themeOverrideTest` should be a non-async function which returns an anonymous async function which runs the actual test logic. Then the `it` statements would just be `it('...', themeOverrideTest(...));`

Sounds great, updated here : [667c925](https://github.com/sonatype/sonatype-react-shared-components/pull/808/commits/667c9254abec3c6cccb7e54e9204c424e012c988)